core/primitives/transaction_validity.hpp

```diff
@@ -62,8 +61,7 @@ namespace kagome::primitives {
      * A non-empty list signifies that some other transactions which provide
      * given tags are required to be included before that one.
      */
-    std::vector<Transaction::Tag>
-    requires;
+    std::vector<Transaction::Tag> requires;
```

Comment:
Revert, or disable clang-format
```suggestion
    // clang-format off
    std::vector<Transaction::Tag> requires;
    // clang-format on
```

Reply:
```clang-format off``` usefull for self-formatted comments or for example self-formatted tables, but not for this.

Reply:
So the problem in formatting just in clang-format version